### DISCUSSION TITLE
How to clean install to UE5?

### DISCUSSION THREAD

**Discussion by tstkenny** [Sixze/ALS-Refactored#183] [Q&A]
As from readme.md:

1.    Clone the repository to your project's Plugins folder, or download the latest release and extract it to your project's Plugins folder.
2.    Merge the DefaultEngine.ini and DefaultInput.ini files from the plugin's Config folder with files from your project's Config folder.
3.    Recompile your project.

May I ask how to "recompile" the project in UE5 ?    Re-opening the project will fail and UE5 exits immediately.

Thank you.

**Comment by ameaninglessname** [comment]
[you can refer to this](https://docs.unrealengine.com/5.0/en-US/building-unreal-engine-from-source)
